app/models/person.rb

```diff
@@ -113,4 +113,8 @@ def phone
   def notify_of_change?(person_responsible)
     EmailAddress.new(email).valid_address? && person_responsible.try(:email) != email
   end
+
+  def is? person
+    person && person.email == self.email
```

Comment:
Docs say ActiveRecord's `==` returns true if objects are of the [same type and have the same `id`](http://api.rubyonrails.org/classes/ActiveRecord/Core.html#method-i-3D-3D). So `==` will work for comparing people. Even if `object_id` is different it will still return true if `id` is the same.

I'll make this change.